ci_scripts/shared-prelude-context/nvm-use.sh

```diff
@@ -0,0 +1,4 @@
+#/bin/bash
+set -xe
+
+nvm install v10
```

Comment:
The v10 part seems like to easy to get out of sync with the travis.yml file.  Can this get the node major version from a travis environment var?  would that be a good idea?  (same in install-nvm-node.sh but it wouldn't let me comment on that line)

Reply:
We're not only using this file from the travis environment, so I would prefer to have the travis environment depend on this, but this would slow down the ci. So I decided to let travis to this thing, we're not changing this very often so it's fine in my opinion. But sure, it's redundant.